### DISCUSSION TITLE
Strange translation

### DISCUSSION THREAD

**Discussion by akseonowww** [phcode-dev/phoenix#1805] [General]
Hello! I wanted to look for a replacement for Sublime Text ala a new notepad and came across you. I like the interface design and the ability to see the result of the code immediately (I'm talking about js). 

But I was surprised at how strangely the line break function was translated - "Заворачивать строки". Please correct it to "Перенос строки" so it's clearer for Russian speakers. 

Thanks for your work.

![image](https://github.com/user-attachments/assets/b95d7c7f-c592-412b-b572-3613ea6d7880)

**Comment by abose** [comment]
Thanks for the fix @akseonowww . We use AI translations and rely on community to provide expert translations. If you do find any other text that need translations, please update here.

**Reply by abose** [reply]
Fixed via https://github.com/phcode-dev/phoenix/pull/1816
Closing as fixed
